src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -36,14 +52,41 @@
     // Matches exactly one of the keywords
     public static final Pattern KEYWORDS_ARGS_FORMAT = Pattern.compile(KEYWORDS_ARGS_FORMAT_BUILDER.toString());
     
-    private final String sortKey;
+    private final String keyExtractorName;
     
     public SortCommand(String key) {
-    	sortKey = key;
+    	keyExtractorName = getKeyExtractorName(key);
+    }
+    
+    public String getKeyExtractorName(String key) {
+    	try {
+    		return SORTKEY_METHOD_NAMES[Arrays.asList(KEYWORDS).indexOf(key)];
+    	} catch (ArrayIndexOutOfBoundsException e) {
+    		return "";
+    	}
```

Comment:
The indentation of this file is not consistent. Try fix the indentation here.

Reply:
As pointed out by Yuntong, there are a lot of indentation issues in your files. Remember that basic indentation should be **4 spaces** and indentation for wrapped lines should be **8 spaces**